data/json/mapgen/nested/light_industry.json

```diff
@@ -0,0 +1,324 @@
+        "|..............................................3",
+        "|..............................................3",
+        "|.........&........................&...........3",
+        "|..............................................3",
+        "|..................X........:..................3",
+        "|..............................................3",
+        "|..........................................3---3",
+        "|............=.............................3   .",
+        "|..........................................3    ",
+        "|----------------------------------------5-3    ",
+        ".    .    .    .  .    .         .         .    ",
+        "                                                ",
+        "                                                ",
+        "                                                ",
+        "                                                ",
+        "                                                ",
+        "                                                ",
+        "                                                "
+      ],
+      "palettes": [ "roof_palette" ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ [ "s_lightindustry_00_roof", "s_lightindustry_01_roof" ] ],
+    "object": {
+      "fill_ter": "t_flat_roof",
+      "rows": [
+        "                                                ",
+        "                                                ",
+        "                                                ",
+        "                                                ",
+        "                                                ",
+        "                                                ",
+        "                                      |22222223 ",
+        " |222222222223                        |.......3 ",
+        " |...........3                        |.......3 ",
+        " 5...........3             |2222222222|.......33",
+        "||...........3  |2223      |...................3",
+        "|............322|...3222222|....X....:.........3",
+        "|..............................................3",
+        "|..............................................3",
+        "|..............................................3",
+        "|..............................................3",
+        "|..........&.............................=.....3",
+        "|.................:............................3",
+        "|............................&.................3",
+        "|....X.........................................3",
+        "|..............................................3",
+        "|.....................................5--------3",
+        "|.....................=...............3         ",
+        "|-------------------------------------3         "
+      ],
+      "palettes": [ "roof_palette" ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "light_industy_nest_zombies",
```

Comment:
Does this nested need a palette?

Reply:
I think? Looked at the evac shelter mapgen to check.

I don't know exactly how to implement this though. Do I just ass the zombie spawns?